src/createServer.js

```diff
@@ -1,3 +1,36 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleErrors } = require('./handleErrors');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const originalText = req.url.split('?').slice(0, 1).join('').slice(1);
+    const queryString = req.url.split('?').slice(1).join('');
+    const params = new URLSearchParams(queryString);
+    const targetCase = params.get('toCase');
+    const errors = handleErrors(originalText, targetCase);
+
+    if (errors.length) {
+      res.statusCode = 400;
+      res.statusMessage = 'Bad request';
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify({ errors }));
+    } else {
+      res.statusCode = 200;
```

Comment:
You have duplicated code here and on 14-17 lines. We can create a function with arguments to avoid duplication. That function just receives status code, message and data for JSON.stringify into arguments. It will be easily to use and reuse this function in future.